xpcPlugin/MessageHandlers.h

```diff
@@ -54,6 +57,14 @@ namespace XPC
 
 		static void HandleXPlaneData(const Message& msg);
 		static void HandleUnknown(const Message& msg);
+		
+		static int CamFunc( XPLMCameraPosition_t * outCameraPosition, int inIsLosingControl, void *inRefcon);
+		
+		struct camera_properties{
+			double loc[3];
+			float direction[3];
```

Comment:
Is `direction` used? I don't see it anywhere else in the PR changes...

Reply:
I have included it since it is a controllable property, which is currently being calculated by the code. But someone might want to set the cam orientation by hand and then these fields can be used. Have added that option to the camera call-back.